src/screens/novel/components/NovelScreenList.tsx

```
@@ -480,6 +478,7 @@ const NovelScreenList = ({
     [lastRead],
   );
 
+  console.log('rendering chapter', fetching, chapters.length);
```

ESLint

[error] Unexpected console statement.

(no-console)